web-client/src/views/Public/TrialSessions/PublicTrialSessionsTable.tsx

```diff
@@ -0,0 +1,110 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
These files below are just moved from `TrialsSessions` to `TrialSessions`, along with commenting in any links to `PublicTrialSessionDetails`.